content-src/selectors/selectors.js

```diff
@@ -5,7 +5,7 @@ const {assignImageAndBackgroundColor} = require("selectors/colorSelectors");
 
 const SPOTLIGHT_LENGTH = module.exports.SPOTLIGHT_LENGTH = 3;
 const TOP_SITES_LENGTH = module.exports.TOP_SITES_LENGTH = 6;
-const MIN_HIGHLIGHTS_LENGTH = module.exports.MIN_HIGHLIGHTS_LENGTH = 5;
+const HIGHLIGHTS_LENGTH = module.exports.HIGHLIGHTS_LENGTH = 12;
```

Comment:
I know this seems super redundant, can we call this `BOTTOM_HIGHLIGHTS_LENGTH` and add another line that says 
```
const TOP_HIGHLIGHTS_LENGTH = SPOTLIGHT_LENGTH;
```